src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,204 @@
+/** @jsxImportSource @emotion/react */
+import React, { useMemo } from 'react';
+import BigNumber from 'bignumber.js';
+import noop from 'noop-ts';
+import { Typography } from '@mui/material';
+import {
+  FormikTokenTextField,
+  LabeledProgressBar,
+  FormikSubmitButton,
+  ConnectWallet,
+  Icon,
+  TokenTextField,
+} from 'components';
+import { useTranslation } from 'translation';
+import useConvertToReadableCoinString from 'hooks/useConvertToReadableCoinString';
+import { format } from 'utilities/common';
+import { AmountForm, ErrorCode } from 'containers/AmountForm';
+import PLACEHOLDER_KEY from 'constants/placeholderKey';
+import { VRT_ID, XVS_ID } from '../constants';
+import { useStyles } from '../styles';
+
+interface ConvertProps {
+  xvsTotalWei: BigNumber;
+  xvsToVrtRate: BigNumber;
+  vrtLimitUsedWei: BigNumber;
+  vrtLimitWei: BigNumber;
+  vrtConversionEndTime: string | undefined;
+}
+
+type FormatI18nextRelativetimeValuesReturn =
+  | {
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHoursAndMinutes';
+      realtiveTimeFormatValues: { hours: number; minutes: number };
+    }
+  | {
+      relativeTimeTranslationKey:
+        | 'convertVrt.remainingTimeDays'
+        | 'convertVrt.remainingTimeHours'
+        | 'convertVrt.remainingTimeMinutes';
+      realtiveTimeFormatValues: { count: number };
+    }
+  | {
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMissing';
+      realtiveTimeFormatValues: { count: string };
+    };
+// Don't remove this comment, it allows for translation keys set by this function to be collected
+// t('convertVrt.remainingTimeDays_one') t('convertVrt.remainingTimeDays_other') t('convertVrt.remainingTimeHours_one') t('convertVrt.remainingTimeHours_other')
+// t('convertVrt.remainingTimeHoursAndMinutes') t('convertVrt.remainingTimeMinutes') t('convertVrt.remainingTimeMinutes_other') t('convertVrt.minutes_one')
+// t('convertVrt.minutes_other') t('convertVrt.hours_one') t('convertVrt.hours_other') t('convertVrt.days_one') t('convertVrt.days_other')
+const formatI18nextRelativetimeValues = (
+  vrtConversionEndTime: string | undefined,
+): FormatI18nextRelativetimeValuesReturn => {
```

Comment:
Why not just take a `Date` param and return the formatted time as a string?

Reply:
We can do that I started with a string because that is it is returned. I'd like to work through this function on [PR 422](https://github.com/VenusProtocol/venus-protocol-interface/pull/422) because that is where I implemented it based on the unix time stamp and not a relative millisecond time like I originally assumed. In this PR it looks like it picked up some changes but not all.